### PR TITLE
Add blocking

### LINKED ISSUE #9: Remove fences from linestring
Remove the `barrier` key from the linestring OSM query. Specifically the `barrier=fence` gives results that make gated communities appear "Off-network" which should only apply to blocks that have no access to street networks.

Example:
`NAM.5.10_1_759`
Link to a gated community -- this is coded off network:
https://www.openstreetmap.org/edit#map=19/-22.56488/17.07176

https://mn-explorer.vercel.app/map#15.75/-22.565173/17.071801

image
image

### PR DESCRIPTION
**Pull request by manmartgarc**
# Description

Adds the ability to block normalized geometries. This is the main functionality of the package and its subject to change in upcoming versions.

The main idea is that from two files for a given country:

1. The land mass
2. The linestrings

We can generate block geometries for the entire country.

Closes #9 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Fixed a couple of unit tests that broke due to some changes. The `blocker.py` module is currently not tested and unit tests are needed.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made sure to bump the version within `pyproject.toml` respectively
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/geopull/extractor.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def _extract_linestring(self, pbf: PBFFile) -> GeoJSONFeatureFile:
             attributes=["type", "id", "version", "changeset", "timestamp"],
             include_tags=[
                 "natural!=coastline,reef",
-                "barrier",
+                "barrier=cityt_wall,ditch",
```
Comment (nmarchio) [inline src/geopull/extractor.py:86]: mis-spelling @manmartgarc 

Include:
"barrier=city_wall,ditch"

File: tests/test_extractor.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def test_extract_linestrings(
             attributes=["type", "id", "version", "changeset", "timestamp"],
             include_tags=[
                 "natural!=coastline,reef",
-                "barrier",
+                "barrier=cityt_wall,ditch",
```
Comment (nmarchio) [inline tests/test_extractor.py:59]: again mis-spelling

**Comment by nmarchio** [comment]
This looks good Manuel -- thanks for all your hard work pushing this big PR through!

File: src/geopull/extractor.py (hunk 1 above)
Comment (manmartgarc) [reply src/geopull/extractor.py:86]: Nice catch, pushed fix

**Pull request merged**